dev_guide/builds.adoc

```diff
@@ -5,19 +5,40 @@
 :icons:
 :experimental:
 :toc: macro
+:toclevels: 3
 :toc-title:
 :prewrap!:
 
 toc::[]
 
 == Overview
-A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build] is a process of creating
-runnable images to be used on OpenShift. There are three build strategies:
+
+A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build]
+is a process of creating runnable images to be used on OpenShift.
+
+Build configurations are characterized by strategy and sources. The strategy
```

Comment:
Are *build* and *build configuration* synonyms?
If so, suggest: "Each build, also known as a *build configuration*, is characterized...".
If not, we need a sentence to relate these concepts (and also reformulate w/ "Each...").